plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/complete-onboarding-wizard.spec.js

```diff
@@ -4,7 +4,7 @@ const { storeDetails } = require( '../../test-data/data' );
 const { api } = require( '../../utils' );
 const { features } = require( '../../utils' );
 
-// Skipping Onbaording tests when the core-profiler is enabled.
+// Skipping Onboarding tests when the core-profiler is enabled.
```

Comment:
I saw these typos while in the area and decided I should just fix them. The formatting is a result of prettier.